content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -14,14 +14,14 @@ Defines a [data source](/concepts/components/prisma-schema/data-sources) <span c
+| `provider`          | **Yes**  | String (`postgresql`, `mysql`, `sqlite`, `sqlserver`, `mongodb`, `cockroachdb`) | Describes which data source connectors to use.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `url`               | **Yes**  | String (URL)                                                                    | Connection URL including authentication info. Most connectors use [the syntax provided by the database](/reference/database-reference/connection-urls#format).                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `shadowDatabaseUrl` | No       | String (URL)                                                                    | Connection URL to the shadow database used by Prisma Migrate. Allows you to use a cloud-hosted database as the shadow database.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `directUrl`         | No       | String (URL)                                                                    | Connection URL for [direct connection to the database](/data-platform/data-proxy/prisma-cli-with-data-proxy#set-a-direct-database-connection-url-in-your-prisma-schema).<br /><br />If you use a connection pooler URL in the `url` argument (for example, if you use the Data Proxy or pgBouncer), Prisma CLI commands [that require a direct connection to the database](/data-platform/data-proxy/prisma-cli-with-data-proxy#prisma-cli-commands-that-require-a-direct-database-connection) use the URL in the `directUrl` argument. <br/> <br/> The `directUrl` property is supported by Prisma Studio from version 5.1.0 upwards. |
```

Comment:
@Jolg42, Let me know if this captures your suggestion.

Reply:
It looks good but what about `content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx`? Because it actually is more valuable there, as we tell users that `prisma studio` "require a direct connection:".
And would not work with an earlier version.
<img width="685" alt="Screenshot 2023-07-31 at 14 44 43" src="https://github.com/prisma/docs/assets/1328733/94ebddd0-ae4a-4395-b082-a97ce40f61fb">

Reply:
Thanks for catching that. I've added a note on that page